common.cmake

```diff
@@ -44,6 +44,10 @@ function(nrfxlib_calculate_lib_path lib_path)
       set(arch_soc_dir ${arch_soc_dir}_cpuapp)
     elseif(DEFINED CONFIG_SOC_NRF5340_CPUNET)
       set(arch_soc_dir ${arch_soc_dir}_cpunet)
+    elseif(DEFINED CONFIG_SOC_NRF54L15_ENGA_CPUAPP)
+      set(arch_soc_dir ${arch_soc_dir}_cpuapp)
+    elseif(DEFINED CONFIG_SOC_NRF54H20_CPURAD)
+      set(arch_soc_dir ${arch_soc_dir}_cpurad)
```

Comment:
With this change in place, I believe the paths to nRF54H20 libraries: https://github.com/nrfconnect/sdk-nrfxlib/tree/main/nrf_802154/sl/sl/lib/nrf54h20 should have been updated too.